.travis.yml

```diff
@@ -30,7 +30,7 @@ matrix:
     - env: RUNTIME=3.6 TOOLKITS="null"
     - env: RUNTIME=3.6 TOOLKITS="pyqt pyqt5 pyside2"
     - env: RUNTIME=3.6 TOOLKITS="wx"
-    - env: RUNTIME=3.6 TOOLKITS="pyqt5" TRAITS_REQUIRES="^=6.0"
```

Comment:
Could this be a separate PR please? It would be good if we see more obviously at what point we stopped testing against Traits 6.0. It needs to be prominent. Also if we are shuffling things around between pyface, envisage and apptools, whereas the next release of envisage and apptools should still support Traits 6.0, we may not be able to drop Traits 6.0 support on Pyface this soon.

Reply:
Oops, this is in traitsui. Anyway, same comment applies here too as envisage and apptools depend on traitsui.

Reply:
reverted.